tests/test_models/test_detectors/test_deformable_detr.py

```diff
@@ -0,0 +1,159 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from unittest import TestCase
+
+import torch
+from mmengine.config import ConfigDict
+from mmengine.structures import InstanceData
+
+from mmdet.models.detectors.deformable_detr import DeformableDETR
+from mmdet.structures import DetDataSample
+from mmdet.utils import register_all_modules
+
+
+class TestDeformableDETR(TestCase):
+
+    def setUp(self):
+        register_all_modules()
+
+    def test_detr_head_loss(self):
+        """Tests transformer head loss when truth is empty and non-empty."""
+        s = 256
+        metainfo = {
+            'img_shape': (s, s),
+            'scale_factor': (1, 1),
+            'pad_shape': (s, s),
+            'batch_input_shape': (s, s)
+        }
+        img_metas = DetDataSample()
+        img_metas.set_metainfo(metainfo)
+        batch_data_samples = []
+        batch_data_samples.append(img_metas)
+
+        config = ConfigDict(
```

Comment:
you can just load the corresponding config by the relative path

Reply:
Ok，I will try